rest/config.go

```diff
@@ -331,7 +334,73 @@ func (dbConfig *DbConfig) setup(name string) error {
 		dbConfig.Server = &urlStr
 	}
 
-	return err
+	// If the sync function is referenced from an external file or http(s) endpoint
+	// unlike the inline JavaScript definition, read it from the specified path.
```

Comment:
This comment is too verbose and a bit unclear; it reads as if loadJavaScript is being called only if it's not inline, but that's not the case.  I think these could all just be simplified as "// Load Sync Function" etc., and leave the verbose description on the loadJavaScript method.